Kitodo-DataManagement/src/main/resources/db/migration/V2_40__Change_projectfilegroup_to_folder.sql

```diff
@@ -69,6 +69,50 @@ ALTER TABLE projectFileGroup
 ALTER TABLE projectFileGroup RENAME TO folder;
 
 
```

Comment:
never change an existing migration! (Unless it is breaking)
If you need to migrate stuf, than add a new migration file.

Reply:
I did this here because it fills the path column below. If I do it later you have to adjust the position twice. That does not make sense. I had planned this from the beginning, I only split it into two pull requests so that the individual packages do not get too big to review.

Reply:
Do you think this is okay for once. The rest of the migration is in a separate file below as you see.

Reply:
But the first version is already in the master! This will break the test system because we need to execute flyway-repair if a migration script changes. Regarding the fact, that we are close to a release and elena is testing constantly I would rather not risk this. Please make a new script or you need to wait untill the release is done :(